types/src/v0/traits.rs

```diff
@@ -437,10 +437,7 @@ pub trait SequencerPersistence: Sized + Send + Sync + 'static {
         // between `highest_voted_view` and `leaf.view_number`. This prevents double votes from
         // starting in a view in which we had already voted before the restart, and prevents
         // unnecessary catchup from starting in a view earlier than the anchor leaf.
-        let mut view = max(highest_voted_view, leaf.view_number());
-        if view != ViewNumber::genesis() {
-            view += 1;
-        }
```

Comment:
Without this, we start in the view we last voted, not after it. So it seems like we are no longer preventing double votes either for consensus or DA. Just want to be sure we're ok with that as a temporary fix, and that we have something better in the pipeline.

Reply:
Yeah it does introduce this, but it's ok for now, I don't think there is any exploit of the potential for double vote